charts/datalayer-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Datalayer Operator
 name: datalayer-operator
 version: 1.0.4
```

Comment:
```suggestion
version: 1.0.5
```

Reply:
@fcollonval  Just pushed a bump and a doc update. Closing this comment as resolved